danPortfolio.html

```diff
@@ -0,0 +1,88 @@
+<html>
+
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0"> 
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	<link rel="stylesheet" type="text/css" href="danPortfolio.css">
+	
+</head>
+
+<body>
+	
+	<nav>
+		<div class="container-for-nav-buttons">
+			<a href="#about" class="three-nav-buttons">about</a>
+			<a href="#portfolio" class="three-nav-buttons">portfolio</a>
+			<a href="#contact" class="three-nav-buttons">contact</a>
+		</div>
+		<a href="#" class="download-cv">download CV</a>
+	</nav>
+	
+	<section class="intro" id="section-with-windows-image">
+		<div class="box container-for-image-and-text-in-intro">
+			<img id="windows-image" src="https://images.adsttc.com/media/images/61e5/71ac/bcfa/3101/659e/e10c/newsletter/bdh-40.jpg?1642426848" alt="windows" width=100%>
+			<article class="brief-intro">
+				<h3>Hello there!</h3>
+				<h1>Welcome to Dan's portfolio</h1>
+				<p>I write code</p>
+			</article>
+		</div>
+	</section>
+		
+	<section class="about" id="about">
+		<div class="box container-for-image-and-text-in-about">
+			<article class="container-for-text-in-about">
+				<h2>About</h2>
+					<p>Hi, I'm Dan, a full-stack programmer and graduate of i0 Academy, based currently just outside the beautiful city of Bath.</p>
+					<p>Feel free to browse the projects below to gauge my programming experience.</p>
+					<p>Outside coding, I enjoy playing the guitar, exercising, and reading.</p>  
+			</article>
+			<div class="container-for-image-in-about">
+				<img id="photo-of-me" img src="https://lh3.googleusercontent.com/a/AEdFTp6WY4Ej7I-rAKYjfmsA3DnIcXg3Ad3tTjwqu6sARQ=s576-p-no" alt="a photo of Dan" width="175px" height="#" style="#">
+			</div>
+		</div>
+	</section>
+
+	<section class="portfolio" id="portfolio">
+		<div class="box">
+			<h2>Portfolio</h2>
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
+				<h4>Project title</h4>
+				<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+				<article class="project-description">In this project, we etc.</article>
+			</div>
+			<div class="project-examples">
+					<h4>Project title</h4>
+					<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+					<article class="project-description">In this project, we etc.</article>
```

Comment:
I think you want to make the text longer here, to help you correctly format this element in terms of text size